mne/io/proj.py

```diff
@@ -100,7 +100,8 @@ def add_proj(self, projs, remove_existing=False, verbose=None):
             self.info['projs'] = projs
         else:
             self.info['projs'].extend(projs)
-
+        self.info['projs'] = _uniquify_projs(self.info['projs'],
+                                             check_active=False, sort=False)
```

Comment:
Can you comment (in the code) on why we can safely do `check_active=False` here? I think it's because, if they are already active in `info` they *won't* get replaced by the new projs (whether active or inactive), but want to make sure